tests/cypress/page-object/pageBuilder.ts

```diff
@@ -0,0 +1,14 @@
+import {BasePage} from '@jahia/cypress';
+
+export class PageBuilder extends BasePage {
```

Comment:
Not sure to understand why a new class ? Why not reusing and returning a ContentEditor instance ?
Like it's the case for the editPage: https://github.com/Jahia/jcontent/blob/master/tests/cypress/page-object/pageComposer.ts#L133-L139
```
 editPage(title: string): ContentEditor {
        cy.iframe('#page-composer-frame', this.iFrameOptions).within(() => {
            cy.get('#JahiaGxtPagesTab').contains(title).rightclick({force: true});
            cy.get('.pagesContextMenuAnthracite').contains('Edit').click({force: true});
        });
        return new ContentEditor();
    }
```